server/backend/courses/tests/test_urls.py

```
@@ -0,0 +1,48 @@
+    DeleteContentView,
+    EnrollCourseView,
+    EnrolledCoursesView,
+    CourseContentView,
+    InstructorCoursesView,
+    RateCourseView,
+    AddContentAPIView,
+)
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "name, kwargs, expected_view",
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)